en/resources/contributing.md

```diff
@@ -374,3 +376,6 @@ We are currently working on a new version of the security model, the most update
 If you have suggestions on how this process could be improved please submit a
 pull request.
 
+# Contributing to Expressjs.com
```

Comment:
I would suggest adding the id so it doesn't change over time
```suggestion
# Contributing to Expressjs.com
{#website-contributing}
```

Reply:
Can you elaborate on this? You mean put an HTML tag with ID?

This title is ok to change over time and not break anything as long as it stays above the page anchoring LOCAL line below. 
This title itself is used as an id, see line 12 of https://github.com/expressjs/expressjs.com/pull/1683/files/2c43c76dd5c16c915e5ff6274d7861739faf9d08#diff-4d54134218ddd8f3f2eed0865238cfe6169fbe2007a2256a93935548a747d79a

Let me know if you meant something else.

Reply:
Oh I didn't know you could do this, and it doesn't seem to work in markdown proper, but only when rendered through to HTML. Since the header auto generates an id as shown in that link above, by changing it to `# Contributing to Expressjs.com {#expressjs-website-contributing}` it replaces the ID the written ID. I made this change.

Is this what you were looking for? Was it just to make it more clear that this section was for website contributing?

Reply:
Yeah, that was it.